Makefile

```diff
@@ -464,7 +464,7 @@ test-e2e-ginkgo: test-e2e-setup
 	--ginkgo.timeout=2h
 
 .PHONY: test-e2e
-test-e2e: volsync-install test-e2e-ginkgo
```

Comment:
also remove command in makefile?

Reply:
Removing volsync everywhere is a (soon) future task. In this topic branch we were doing the minimum to get the new velero 1.12 code working w/ oadp. There's still more work to do to clean it all up, but we'd like to get 1.12 velero on oadp master sooner rather than later so it gets more testing and exposure.

Reply:
right, undertood